packages/badge/src/react/__stories__/index.story.tsx

```diff
@@ -1,28 +1,37 @@
-import { ValueOf } from '@pluralsight/ps-design-system-util'
-import { storiesOf } from '@storybook/react'
+import { Meta, Story } from '@storybook/react/types-6-0'
 
-import React from 'react'
+import * as React from 'react'
```

Comment:
is the `* as` related to your ESM work?

Reply:
yeah, essentially. This is the correct import style in esm land. Otherwise, have to add `allowSyntheticDefaultImports` to tsconfig.